Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.9
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
```

Comment:
Either this one is not needed, if we use `COPY . .` for all files anyway.
Or `COPY . .` need to list just `radoneye-reader.py`

Reply:
It's a caching optimization - causes pip install to only run when the requirements change

Reply:
ok